one-container/docker-compose.yaml

```diff
@@ -31,5 +31,6 @@ services:
     volumes:
       - etc_pihole-unbound:/etc/pihole:rw
       - etc_pihole_dnsmasq-unbound:/etc/dnsmasq.d:rw
+      - '${UNBOUND_CONFIG_MOUNT}'
```

Comment:
This doesn't appear to be an optional variable/volume config, which now means all users need to mount custom unbound configs (most users don't need this).

Reply:
We can just comment that out, as you have done [here] (https://github.com/chriscrowe/docker-pihole-unbound/blob/main/one-container/docker-compose.yaml#L19)